pkgs/by-name/fu/furtherance/package.nix

```diff
@@ -3,61 +3,71 @@
   stdenv,
   fetchFromGitHub,
   rustPlatform,
-  appstream-glib,
-  cargo,
-  desktop-file-utils,
-  glib,
-  libadwaita,
-  meson,
-  ninja,
+  fontconfig,
+  libxkbcommon,
+  openssl,
   pkg-config,
-  rustc,
-  wrapGAppsHook4,
-  dbus,
-  gtk4,
-  sqlite,
+  xorg,
+  vulkan-loader,
 }:
-
-stdenv.mkDerivation (finalAttrs: {
+rustPlatform.buildRustPackage (finalAttrs: {
   pname = "furtherance";
-  version = "1.8.3";
+  version = "25.3.0";
 
   src = fetchFromGitHub {
-    owner = "lakoliu";
+    owner = "unobserved-io";
     repo = "Furtherance";
-    rev = "v${finalAttrs.version}";
-    hash = "sha256-TxYARpCqqjjwinoRU2Wjihp+FYIvcI0YCGlOuumX6To=";
+    rev = finalAttrs.version;
+    hash = "sha256-LyGO+fbsu16Us0+sK0T6HlGq7EwZWSetd+gCIKKEbkk=";
   };
 
-  cargoDeps = rustPlatform.fetchCargoVendor {
-    inherit (finalAttrs) pname version src;
-    hash = "sha256-SFp9YCmneOll2pItWmg2b2jrpRqPnvV9vwz4mjjvwM4=";
-  };
+  cargoHash = "sha256-j/5O40k12rl/gmRc1obo9ImdkZ0Mdrke2PCf6tFCWIo=";
 
   nativeBuildInputs = [
-    appstream-glib
-    desktop-file-utils
-    meson
-    ninja
+    # appstream-glib
+    # desktop-file-utils
+    # rustPlatform.cargoSetupHook
+    # cargo
+    # rustc
+    # wrapGAppsHook4
```

Comment:
I think it will be cleaner to get rid of these commented out lines.

Reply:
Oops, I forgot to remove them, it’s done now